ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -142,6 +142,11 @@ def pytest_addoption(parser):
         dest='upgrade_ocs_version',
         help="ocs version to upgrade (e.g. 4.3)"
     )
+    parser.addoption(
+        '--upgrade-ocp-version',
+        dest='upgrade_ocp_version',
+        help='ocp version to upgrade to (e.g. 4.5)'
```

Comment:
The info should look like more similar to --ocp-version parameter where we accept -ga versions as well.
So I would add examples like we have in there:
```
        4.2: for nightly 4.2 OCP build
        4.2-ga: for latest GAed 4.2 OCP build
```

Reply:
Done